examples/examples/fmt/yak_shave.rs

```diff
@@ -1,92 +1,57 @@
-#![deny(rust_2018_idioms)]
-use tracing::{debug, error, info, span, trace, warn, Level};
-
 use std::{error::Error, fmt};
+use tracing::{debug, error, info, span, warn, Level};
```

Comment:
I combined these two imports and reduce the number of levels used in this example from four (trace, debug, info, and warn) to three (debug, info, and warn). I can undo this, but I'd like to also have an example with _all_ the levels available.

Reply:
This is fine with me — I notice that `log`'s [version of this example](https://github.com/rust-lang/log#in-libraries) also only uses 3 levels. 

I definitely agree it's important to show all the levels, though...we could just have an example like 
```rust
fn main() {
    tracing::error!("SOMETHING IS SERIOUSLY WRONG!!!");
    tracing::warn!("important informational messages; might indicate an error");
    tracing::info!("general informational messages relevant to users");
    tracing::debug!("diagnostics used for internal debugging of a library or application");
    tracing::trace!("very verbose diagnostic events");
}
```

Reply:
I like that idea; I'll add that as an example.

Reply:
added.